### PR TITLE
Removed link to MATLAB as the link does not work anymore.

### PR DESCRIPTION
**Pull request by jonasbjork**
## What does this PR do?
I've removed link to MATLAB pdf file as the file is missing now.

### CODE REVIEW

**Comment by davorpa** [comment]
Scope PR: https://github.com/EbookFoundation/free-programming-books/pull/6229/commits/bac91c6aa1c94f3134bc8e00c3f8f39930cfd3b1

**Comment by eshellman** [comment]
In other words, start from a fresh branch because the PR had twice the changes you meant.

File: books/free-programming-books-se.md [hunk 1]
```diff
@@ -15,11 +15,6 @@
 * [Programmera spel i C++ för nybörjare](https://sv.wikibooks.org/wiki/Programmera_spel_i_C%2B%2B_f%C3%B6r_nyb%C3%B6rjare) - Wikibooks
 
 
-### MATLAB
-
-* [Introduktion till MATLAB](https://www.liber.se/plus/E470523401.pdf) (PDF)
-
-
```
Comment (davorpa) [inline books/free-programming-books-se.md:22]: An alternative hosted at LINKÖPINGS UNIVERSITET: https://www.cvl.isy.liu.se/education/undergraduate/TSKS08/matlab-1/resurser/


Apply this patch to restore resource
```suggestion
### MATLAB

* [Introduktion till MATLAB (2004)](https://www.cvl.isy.liu.se/education/undergraduate/TSKS08/matlab-1/Matlabintro_sve.pdf) - Liber AB, Lennart Harnefors, Johnny Holmberg, Joop Lundqvist (PDF)


```

**Review by davorpa — CHANGES_REQUESTED** [review]
ISO language code for swedish is `sv` see #6036.

Sorry for the conflicts due to it. Merge with upstream after apply suggestions

File: books/free-programming-books-se.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-se.md:22]: Suggestion applied at #6595

**Comment by davorpa** [comment]
Conflicts solved at #6595

**Comment by davorpa** [comment]
Duplicate of #6595

**Pull request closed without merging**